### PR TITLE
OADP v1.1.3 updates

### PR DESCRIPTION
**Pull request by hhpatel14**
Update lab to version v1.1.3 and address issues filed by users: 

-  https://github.com/redhat-cop/agnosticd/issues/6265

- [Lab feedback by User](https://docs.google.com/document/d/12tg2_arZ2zspRD8V1mGNLbTTr9hUjphztTkZFv6oRoM/edit)

### CODE REVIEW

**Comment by kaovilai** [comment]
1.1.4 is out now.

**Review by kaovilai — APPROVED** [review]

File: oadp/bookbag/workshop/content/Conclusion.adoc [hunk 1]
```diff
@@ -16,6 +16,6 @@
 |===
 |Type |Link
 |Source Code |https://github.com/openshift/oadp-operator
-|Velero Documentation |https://velero.io/docs/v1.7
+|Velero Documentation |https://velero.io/docs/v1.9/
```
Comment (weshayutin) [inline oadp/bookbag/workshop/content/Conclusion.adoc:19]: might as well use 1.11 so we don't forget

Comment (kaovilai) [reply oadp/bookbag/workshop/content/Conclusion.adoc:19]: 1.11 will be for https://github.com/migtools/labs/pull/68

Comment (hhpatel14) [reply oadp/bookbag/workshop/content/Conclusion.adoc:19]: Already pushed 1.11 to #68

**Review by weshayutin — APPROVED** [review]

**Pull request merged**